streamer/src/nonblocking/quic.rs

```diff
@@ -811,8 +844,15 @@ async fn handle_connection(
         stats.total_streams.load(Ordering::Relaxed),
         stats.total_connections.load(Ordering::Relaxed),
     );
+    connection.set_max_concurrent_uni_streams(max_uni_streams);
+    if let Some(receive_window) = receive_window {
```

Comment:
Any benefit of moving receive_window setting to this function?

Reply:
It encapsulates better to put QOS related config for connections in one place (receive window and max concurrent uni streams.